src/components/MarkerProps.tsx

```diff
@@ -71,6 +71,49 @@ export default interface MarkerProps {
    */
   selectedGlyphImage?: object | undefined;
 
+  /**
+   * The desired dimensions of the annotation, in CSS pixels.
+   * @see {@link https://developer.apple.com/documentation/mapkitjs/mapkit/annotation/2973833-size}
+   * @example `{width: 100, height: 100}`
+   */
+  size?: { width: number; height: number };
```

Comment:
The documentation of MapKit JS doesn’t make it clear, but I just found out during testing that it is only possible to set the size of non-marker annotations. We should remove `size` from `Marker`/`MarkerProps` as it is not usable.

![[MapKit] The `size` property of a marker annotation cannot be set.](https://github.com/Nicolapps/mapkit-react/assets/7029582/346b9755-4e63-4d4a-9d34-d5e8af8d21bf)

Reply:
Thank you for your testing, I removed it. Talking about it, it does indeed kind of makes sense, though I also wasn't able to find anything about it in the documentation or on the web.